client/src/components/DataFiles/DataFilesProjectMembers/DataFilesProjectMembers.module.scss

```diff
@@ -50,7 +42,7 @@
   /* owner */
   th:nth-child(2),
   td:nth-child(2) {
-    min-width: 10%;
+    width: 10%;
```

Comment:
This change occurs because flex is no longer used for table layout, so the `table-layout: fixed` we had now kicks in. (See [`InfiniteScrollTable.scss`](https://github.com/TACC/Core-Portal/blob/task/FP-385-section-comp-welcome-msg--sections/client/src/components/_common/InfiniteScrollTable/InfiniteScrollTable.scss) for the relevant code.)